README.md

```diff
@@ -62,6 +63,17 @@ $ make build
 $ ./bin/terrascan
 ```
 
+### Demoing
+If you want to play around with Terrascan without running it locally, you can
+[boot a personal demo copy
+](https://kelda.io/preview-env/?repo=https://github.com/accurics/terrascan.git&composeFiles=deploy/docker-compose.yml&service=terrascan&port=9010)
+from your browser without downloading or setting up anything.
+
+Clicking the
+[link](https://kelda.io/preview-env/?repo=https://github.com/accurics/terrascan.git&composeFiles=deploy/docker-compose.yml&service=terrascan&port=9010)
+boots this repo in the Blimp cloud, and creates a public URL for you to access
+it.
+
```

Comment:
Would you mind adding the curl example of how to interact with the sandbox once it's up?

Reply:
Done! The example terraform file I used didn't have any violations, but if you have a different file that you think would be a better demo I can update it.